pkg/types/baremetal/defaults/platform.go

```diff
@@ -69,4 +74,39 @@ func SetPlatformDefaults(p *baremetal.Platform, c *types.InstallConfig) {
 			p.IngressVIP = vip[0]
 		}
 	}
+
+	if p.ProvisioningInterface == "" {
+		p.ProvisioningInterface = ProvisioningInterface
+	}
+
+	if p.ProvisioningNetworkCIDR == "" {
+		p.ProvisioningNetworkCIDR = ProvisioningNetworkCIDR
+	}
+
+	_, provNet, _ := net.ParseCIDR(p.ProvisioningNetworkCIDR)
+
+	if p.ProvisioningIP == "" {
+		ip, err := cidr.Host(provNet, 3)
+		if err == nil {
```

Comment:
I think this replaces the hard-coded default for ClusterProvisioningIP, and we should probably also calculate BootstrapProvisioningIP?